src/monattack.cpp

```diff
@@ -2048,15 +2029,15 @@ void mattack::dermatik_growth(monster *z, int index)
     z->poly(GetMType("mon_dermatik"));
 }
 
-void mattack::plant(monster *z, int index)
+void mattack::plant( monster *z, int /*index*/ )
```

Comment:
That avoids the compiler warning "unused parameter" (or similar). The parameter needs to be there because it's part of the interface, the caller doesn't know the function will not use the value.

There are some alternatives to that: http://stackoverflow.com/questions/1486904